apix-rest-v1/src/main/java/eu/xenit/apix/rest/v1/nodes/NodesWebscript1.java

```diff
@@ -963,6 +976,21 @@ private void writeNotAuthorizedResponse(AccessDeniedException exception, WebScri
         writeJsonResponse(response, "Not authorised to execute this operation");
     }
 
+    @ExceptionHandler(FileExistsException.class)
+    private void writeFileExistsResponse(FileExistsException fileExistsException, WebScriptResponse response) throws IOException {
+        StringBuilder messagebuilder = new StringBuilder("A file with name ")
```

Comment:
Would you mind adding quotes (`'`) around `fileExistsException.name` to improve readability a little bit in the message?